library/std/src/io/mod.rs

```diff
@@ -2462,6 +2470,35 @@ impl<R: Read> Iterator for Bytes<R> {
             };
         }
     }
+
+    default fn size_hint(&self) -> (usize, Option<usize>) {
```

Comment:
IIRC public traits aren't supposed to have `default` on any of their methods.

Oh, you're not even overriding it, so this seems like it might just be residual from a previous version?

Reply:
Yeah, I'm having trouble getting it to work.

Reply:
It won't build locally, so I keep pushing it for the CI to test. :(

Reply:
I think this should work now

Reply:
> It won't build locally, so I keep pushing it for the CI to test. :(

`./x.py check library/std` should cover these issues and skip most of the more demanding build steps